wherehows-frontend/conf/application.conf

```diff
@@ -92,3 +92,4 @@ lineage.look.back.time = ${?LINEAGE_LOOK_BACK_TIME}
 tracking.piwik.siteid = ${?PIWIK_SITE_ID}
 tracking.piwik.url = ${?PIWIK_URL}
 
+ui.show.ownership.revamp = ${?WHZ_DS_OWNERSHIP_TAB}
```

Comment:
I am sorry, what I really meant it to omit this line. This is a minor point. Otherwise, LGTM

Reply:
Shipit